OCaml.tmLanguage

```diff
@@ -1626,7 +1626,7 @@
 			<array>
 				<dict>
 					<key>begin</key>
-					<string>(val!?)(\s+mutable)?\s+([a-z_][a-zA-Z0-9_']*|\([=&lt;&gt;@^&amp;+\-*/$%|][|!$%&amp;*+./:&lt;=&gt;?@^~-]*\))\s*(:)</string>
+					<string>(val!?)(\s+mutable)?\s+([a-z_][a-zA-Z0-9_']*|\([=&lt;&gt;@^&amp;+\-*/$%|][|!$%&amp;*+./:&lt;=&gt;?@^~-]*\))\s*(:?)</string>
```

Comment:
Ah, so we use this style in our mli files:
```
val create
    :  date   : Date.t
```
with the : on the following line, and this just wasn't picking those cases up